chains/73773/chain-information.json

```diff
@@ -0,0 +1,48 @@
+{
+  "chainName": "Swimmer Network Subnet",
+  "description": "Swimmer Network Subnet is home to Crabada, an undersea world filled with fierce fighting Hermit-Crabs called Crabada. The Crabada Idle-Game and marketplace is expected migrate from the C-chain onto the Swimmer Network. Crabada can be bred, and used in games to earn cryptocurrencies with real value.",
```

Comment:
They have a different description in that doc too.

Reply:
This is fine - we're just keeping the same name that we had (I checked with Patrick)